browser/js/common/directives/landing/landing.js

```diff
@@ -32,23 +34,27 @@ app.directive('landing', function(){
 			//.append()- appending a (group) element, not sure why this is needed?
 
 			var svg = d3.select("#home")
+						.append("div")
+						// .classed("svg-container", true)
 					    .append("svg")
-					    .attr("width", width)
-					    .attr("height", height)
-		    		    .call(d3.behavior.zoom()
-		    		    .on("zoom", redraw))
+					    //responsive SVG needs these 2 attributes and no width and height attr
```

Comment:
Its not permanent. im still experimenting with the responsiveness. i just commented it out for now to see how it works without it. we can discuss it tomorrow